client-app/components/molecules/item-price/item-price.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="flex flex-col">
+    <div class="flex space-x-1">
+      <VcPriceDisplay
+        v-if="value?.list?.amount > value?.sale?.amount"
+        class="font-extrabold"
+        :class="priceColorClass"
+        :value="value?.sale"
+      />
+      <VcPriceDisplay v-else class="text-green-700 font-extrabold" :value="value?.list" />
+      <span class="hidden lg:inline" v-t="'common.suffixes.per_item'"></span>
+    </div>
+    <div>
+      <VcPastPriceDisplay
```

Comment:
Do we really need this component? Since we pass text color, size and font weight through classes, why we cannot pass `line-through` class to make price crossed? I think this is useless component in the UI Kit.

Reply:
This component incapsulate logic of showing list/sale price. May be you mean another component. VcPastPriceDisplay? 
For this it is legal note.